src/main/resources/org/jenkinsci/plugins/github/config/GitHubPluginConfig/config.groovy

```diff
@@ -5,15 +5,12 @@ import com.cloudbees.jenkins.GitHubPushTrigger
 def f = namespace(lib.FormTagLib);
 
 f.section(title: descriptor.displayName) {
-    f.entry(title: _("Servers configs with credentials to manage GitHub integrations"),
-            description: _("List of GitHub Servers to manage hooks, set commit statuses etc."),
```

Comment:
Yes, github servers configuration part should be generified in UI and docs. But clicking help buttons is also not always convenient. Try open all collapsed items in configuration page to get review of configurations - you will tire clicking.